clients/pkg/promtail/targets/file/filetargetmanager.go

```diff
@@ -373,7 +395,6 @@ func (s *targetSyncer) sync(groups []*targetgroup.Group, targetEventHandler chan
 			}
 		}
 	}
-	s.droppedTargets = dropped
```

Comment:
moving this line doesn't seem necessary to change the behaviour of keeping or removing a file watcher, so lets not move it in this PR